dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -726,7 +738,14 @@ private WorkflowVersion setupWorkflowFilesForGitHubVersion(Triple<String, Date,
         try {
             final DockstoreYaml12 dockstoreYaml12 = DockstoreYamlHelper.readAsDockstoreYaml12(dockstoreYml.getContent());
             // TODO: Need to handle services; the YAML is guaranteed to have at least one of either
-            final Optional<YamlWorkflow> maybeWorkflow = dockstoreYaml12.getWorkflows().stream().filter(wf -> {
+            List<YamlWorkflow> workflows;
+            if (workflow instanceof OneStepWorkflow) {
```

Comment:
Does this mean one repo can only have either one-step workflows or workflows, but not both?

Reply:
Should be able to have both, but I will double check

Reply:
Yes, a single repo can have both tools and workflows.